Sources/Core/Views/GenericButton/GenericButton.swift

```diff
@@ -0,0 +1,176 @@
+//
+//  GenericButton.swift
+//  PovioKit
+//
+//  Created by Arlind Dushi on 3/17/22.
+//  Copyright © 2022 Povio Inc. All rights reserved.
+//
+
+import SwiftUI
+
+@available(iOS 13, *)
+public class GenericButtonProperties: ObservableObject {
```

Comment:
Maybe it would be better to change this name to be more in SwiftUI style: _GenericButtonViewModel_